docs/DEVICE-CONNECTIVITY.md

```diff
@@ -76,7 +76,7 @@ The nim retains the currently working configuration, plus the following in prior
 1. The most recently received configuration from the controller
 1. The last known working configuration from the controller
 1. Bootstrap config or an override file from a USB stick (if any)
-1. The last resort if so enabled
+1. The last resort if enabled, or (used forcefully) if there is no network config available (e.g. first boot without bootstrap and override)
```

Comment:
"without bootstrap config or config override file" seems more correct to me

Reply:
Probably it is :)
Fixed